src/src/openthread-core-efr32-config.h

```diff
@@ -52,19 +46,27 @@
 #include SL_OPENTHREAD_STACK_FEATURES_CONFIG_FILE
 #endif
 
+#include "board_config.h"
+
 /**
- * @def OPENTHREAD_CONFIG_LOG_OUTPUT
+ * @def OPENTHREAD_CONFIG_CLI_MAX_LINE_LENGTH
+ *
+ * The maximum size of the CLI line in bytes including the null terminator.
  *
- * The efr32 platform provides an otPlatLog() function.
  */
-#ifndef OPENTHREAD_CONFIG_LOG_OUTPUT
-#define OPENTHREAD_CONFIG_LOG_OUTPUT OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED
```

Comment:
Removed because this is already the default value 
https://github.com/openthread/openthread/blob/9bb09e74e2bfddb8e5272cc8ade6e0fa4f8125f0/src/core/config/logging.h#L58-L60